### ISSUE TITLE
Return `relativeLocation` Feature with forecast

### ISSUE THREAD

**Issue by roblabs** [WeatherProvider/NationalWeatherService-Swift#11]
The NWS API is returning `relativeLocation` in empirical testing.  

It looks like its fairly basic to implement (example [on my branch](https://github.com/roblabs/NationalWeatherService-Swift/commit/001a4dee80a4fa1a10432416d93220c201bdbe0e)), but I was wondering what would be the best way to return `relativeLocation` with the `Forecast` object?

The current Swift API is very straightforward.  But how should we return some of the useful data from the `Points` object?  I didn't want to _gum up_ the returning result object with data not in line with the NWS API, so I wanted to see what others thought.

```swift
    nws.forecast(for: location) { result in
        print(result)
        // How to include points.properties.relativeLocation
    }
```

---

Example, 

* Willow Hole, Joshua Tree, CA -  would return a relative location of *"7 miles SW of Twentynine Palms, CA"*
  * https://api.weather.gov/points/34.06843,-116.15310
  * Raw relative location could be: *"11.9 KM at a bearing of 222 degrees from Twentynine Palms, CA"*
  
```json
{
"relativeLocation": {
            "type": "Feature",
            "geometry": {
                "type": "Point",
                "coordinates": [ -116.06552600000001, 34.148313000000002 ]
            },
            "properties": {
                "city": "Twentynine Palms", "state": "CA",
                "distance": {
                    "unitCode": "wmoUnit:m", "value": 11998.621146330999
                },
                "bearing": {
                    "unitCode": "wmoUnit:degree_(angle)", "value": 222
                }
            }
        }
}
```

**Comment by ualch9** [comment]
https://github.com/roblabs/NationalWeatherService-Swift/commit/001a4dee80a4fa1a10432416d93220c201bdbe0e#:
```diff
- public let relativeLocation: MKGeoJSONFeature

+ import GEOSwift
+ public let relativeLocation: Feature
```

IIRC, I didn't want to use GEOSwift.Feature because I didn't want to make this library expose third party framework as public. MKGeoJSONFeature is MapKit, which is okay. 

I wrote this a while ago, and didn't write comments. I'm assuming that MKGeoJSONFeature doesn't cleanly decode from NWS's API, which is why I commented it out. I can take a look at this further.

> [...] gum up the returning result object with data not in line with the NWS API [...]

Most data should be in line with NWS's schema, however, I think it's okay to be slightly different as long as it improves interoperability with Swift STD and Foundation.
